go.sum

```diff
@@ -180,7 +182,7 @@ github.com/enigmampc/btcutil v1.0.3-0.20200723161021-e2fb6adb2a25 h1:2vLKys4RBU4
 github.com/enigmampc/btcutil v1.0.3-0.20200723161021-e2fb6adb2a25/go.mod h1:hTr8+TLQmkUkgcuh3mcr5fjrT9c64ZzsBCdCEC6UppY=
 github.com/envoyproxy/go-control-plane v0.9.4/go.mod h1:6rpuAdCZL397s3pYoYcLgu1mIlRU8Am5FuJP05cCM98=
 github.com/envoyproxy/protoc-gen-validate v0.1.0/go.mod h1:iSmxcyjqTsJpI2R4NaDN7+kN2VEUnK/pcBlmesArF7c=
-github.com/ethereum/go-ethereum v1.9.23/go.mod h1:JIfVb6esrqALTExdz9hRYvrP0xBDf6wCncIu1hNwHpM=
+github.com/ethereum/go-ethereum v1.9.25/go.mod h1:vMkFiYLHI4tgPw4k2j4MHKoovchFE8plZ0M9VMk4/oM=
```

Comment:
go-ethereum is quite a big dependency -- would it be possible to strip it off for the rosetta support?

Reply:
IIRC it's a rosetta dependency

Reply:
yes, correct: https://github.com/coinbase/rosetta-sdk-go/blob/9ba330f0a51ab41c1c8738a040eb2b28894b82d9/go.mod#L13

Reply:
It seems Rosetta SDK only uses secp256k1 signing from it: https://github.com/coinbase/rosetta-sdk-go/blob/9ba330f0a51ab41c1c8738a040eb2b28894b82d9/keys/signer_secp256k1.go#L21 ?
That seems like an overkill

Reply:
@tomtau mind opening up an issue in [the upstream's bugtracker](https://github.com/coinbase/rosetta-sdk-go/issues/new/choose), please?

Reply:
https://github.com/coinbase/rosetta-sdk-go/issues/318